schematic/store/synapse.py

```diff
@@ -43,17 +43,17 @@
 from schematic_db.rdb.synapse_database import SynapseDatabase
 
 
-from schematic.utils.df_utils import update_df, load_df, col_in_dataframe, populate_df_col_with_another_col
```

Comment:
`populate_df_col_with_another_col` function only shows up in df_utils and test_utils, and is not used anywhere else.